cmd/ocsp-responder/main.go

```diff
@@ -267,18 +259,19 @@ func (src *dbSource) Response(ctx context.Context, req *ocsp.Request) ([]byte, h
 		select {
 		case <-ctx.Done():
 			if errors.Is(ctx.Err(), context.Canceled) {
-				src.metrics.ocspLookups.WithLabelValues("canceled").Inc()
+				src.metrics.ocspLookups.WithLabelValues("mysql", "failed", "canceled").Inc()
 			}
-			src.metrics.ocspLookups.WithLabelValues("deadline_exceeded").Inc()
+			src.metrics.ocspLookups.WithLabelValues("mysql", "failed", "deadline_exceeded").Inc()
 			return nil, nil, fmt.Errorf("looking up OCSP response for serial: %s err: %w", serialString, ctx.Err())
 		case primaryResult = <-primaryChan:
 		}
 
 		// Check for error returned from the mysql lookup, return on error.
 		if primaryResult.err != nil {
-			if !errors.Is(primaryResult.err, bocsp.ErrNotFound) {
-				src.metrics.ocspLookups.WithLabelValues("mysql_failed").Inc()
-				src.metrics.sourceUsed.WithLabelValues("error_returned").Inc()
+			if errors.Is(primaryResult.err, bocsp.ErrNotFound) {
+				src.metrics.ocspLookups.WithLabelValues("mysql", "success", "not_found").Inc()
+			} else {
+				src.metrics.ocspLookups.WithLabelValues("mysql", "failed", "error").Inc()
```

Comment:
maybe "other" instead of "error", since the question "what kind of error was it?" isn't readily answered by "error" :P

Reply:
Indeed. I chose "error" because that is the label that increments if it is an error we care about and the user of a dashboard wouldn't really see the redundancy, they would likely see the label and want to know if it is something they should care about or not. That said, I don't have a strong preference and can push a change for "other".

Reply:
Yeah, I'd prefer "other", so it's in the same semantic category as all the other values for this label.

Reply:
Done!

Reply:
Why do we have both `result` and `error` labels? Seems like it would make more sense to have `result`, with values "success", "not found", "canceled", "deadline exceeded", "other".

Reply:
I have pushed another update and got rid of the error labels. The current possible results are  "success", "failed", "not found", "mismatch", "parse_error", "canceled", "deadline exceeded".

I had both `result` and `error` because `not found` is technically a success, but seemed like a good thing to add that it was a success, but not found. It is probably not necessary so the current change set got rid of it.